projects/socket_example/src/x86/main.c

```diff
@@ -0,0 +1,21 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <unistd.h>
+#include "x86_socket.h"
+
+static void prv_handler(X86SocketThread *thread, int client_fd, const char *rx_data, size_t rx_len,
```

Comment:
Nit: Comments? This is pretty self explanatory but we should try to comment any private functions.

Reply:
I'll probably move it to a test - this was just a sample program I was experimenting with.